cpp/src/parquet/statistics.cc

```diff
@@ -458,6 +540,16 @@ std::pair<ByteArray, ByteArray> TypedComparatorImpl<false, ByteArrayType>::GetMi
   return GetMinMaxBinaryHelper<false>(*this, values);
 }
 
+static LogicalType::Type::type LogicalTypeId(const ColumnDescriptor* descr) {
```

Comment:
Not sure the `static` is required (aren't we already in the anonymouns namespace?).